doc/reference/authpolicy.md

```diff
@@ -0,0 +1,151 @@
+| `kubernetesSubjectAccessReview` | [Kubernetes SubjectAccessReview spec](https://docs.kuadrant.io/authorino/docs/features/#kubernetes-subjectaccessreview-authorizationkubernetessubjectaccessreview) | No           | Authorization by Kubernetes SubjectAccessReview. Use one of: `patternMatching`, `opa`, `kubernetesSubjectAccessReview`, `spicedb`.                     |
+| `spicedb`                       | [SpiceDB authorization spec](https://docs.kuadrant.io/authorino/docs/features/#spicedb-authorizationspicedb)                                                       | No           | Authorization decision delegated to external Authzed/SpiceDB server. Use one of: `patternMatching`, `opa`, `kubernetesSubjectAccessReview`, `spicedb`. |
+| _(inline)_                      | [AuthRuleCommon](#authrulecommon)                                                                                                                                  | No           |                                                                                                                                                        |
+
+#### ResponseSpec
+
+| **Field**         | **Type**                                                                                                                                                           | **Required** | **Description**                                                                                                                    |
+|-------------------|--------------------------------------------------------------------------------------------------------------------------------------------------------------------|:------------:|------------------------------------------------------------------------------------------------------------------------------------|
+| `unauthenticated` | [Custom denial status spec](https://docs.kuadrant.io/authorino/docs/features/#custom-denial-status-responseunauthenticated-and-responseunauthorized)               | No           | Customizations on the denial status and other HTTP attributes when the request is unauthenticated. (Default: `401 Unauthorized`)   |
+| `unauthorized`    | [Custom denial status spec](https://docs.kuadrant.io/authorino/docs/features/#custom-denial-status-responseunauthenticated-and-responseunauthorized)               | No           | Customizations on the denial status and other HTTP attributes when the request is unauthorized. (Default: `403 Forbidden`)         |
+| `success`         | [SuccessResponseSpec](#successresponsespec)                                                                                                                        | No           | Response items to be included in the auth response when the request is authenticated and authorized.                               |
+
+##### SuccessResponseSpec
```

Comment:
`SuccessResponseSpec` (h4) is used within `ResponseSpec` (h3). So it's correct.

It's actually `SuccessResponseItem` that should be h5, because it's used only within `SuccessResponseSpec` (h4).

Reply:
I will leave this up to you, if you want to change it. It did seem odd but that could also be how it was rendered in the rich diff.